sense platform/sensors/CSNoiseSensor.m

```diff
@@ -105,149 +111,24 @@ - (void) configureAudioSession
     AVAudioSession *session = [AVAudioSession sharedInstance];
     NSError *activationError = nil;
     NSError *categoryError = nil;
-
+	NSError *modeError = nil;
+	
     // audio session category
     NSString *appAudioSessionCategory = AVAudioSessionCategoryPlayAndRecord;
```

Comment:
@Tubyhes documentation says it silences playback audio. That's not what we want...